libs/gl-plugin/src/node/mod.rs

```diff
@@ -27,6 +27,8 @@ use tokio_stream::wrappers::ReceiverStream;
 use tonic::{transport::ServerTlsConfig, Code, Request, Response, Status};
 mod wrapper;
 pub use wrapper::WrappedNodeServer;
+use::gl_client::bitcoin;
```

Comment:
```suggestion
use gl_client::bitcoin;
```
Did this compile?

Reply:
Local, CI, and [4-line reproduction](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=66cbd9b9f10d046764ebabf1e5b1c671) tell me yes 🙂

I'll remove the extra colons though

Reply:
Removed colons